jcl/src/java.base/share/classes/com/ibm/tools/attach/target/CommonDirectory.java

```diff
@@ -276,6 +276,29 @@ public static int notifyVm(int numberOfTargets) {
 		return IPC.notifyVm(getCommonDirFileObject().getAbsolutePath(), MASTER_NOTIFIER, numberOfTargets);
 	}
 
+	/**
+	 * Open the semaphore, post to it, and close it
+	 * @param numberOfTargets number of times to post to the semaphore
+	 * @param global Use the global semaphore (Windows only)
+	 * @return 0 on success
+	 */
+	public static int notifyVm(int numberOfTargets, boolean global) {
```

Comment:
It seems unnecessary to add new API and keep the old API, just to avoid changing a few callers. By my count there are only 2 places that need to change.

Reply:
Fixed.